okta-hosted-login/src/config.js

```diff
@@ -1,12 +1,14 @@
+/* global window */
 const CLIENT_ID = process.env.CLIENT_ID || '{clientId}';
 const ISSUER = process.env.ISSUER || 'https://{yourOktaDomain}.com/oauth2/default';
 const OKTA_TESTING_DISABLEHTTPSCHECK = process.env.OKTA_TESTING_DISABLEHTTPSCHECK || false;
+const REDIRECT_URI = `${window.location.origin}/implicit/callback`;
```

Comment:
Remembering that these samples are often used as the templates for production apps, how do you feel about adding a comment above this explaining?  Perhaps:
```
// Hardcode to the correct value if you have multiple subdomains or other complex situation
const REDIRECT_URI = `${window.location.origin}/implicit/callback`;
```

Reply:
I think there are very few cases where you would hardcode the entire URL, I think it would also be very rare that the redirect origin would not match the current origin. More likely would be customizing the callback path (which we have hardcoded to `/implicit/callback`). I can imagine a scenario where the callback path is situational, maybe based on device type or client id? But I haven't seen something like this yet.

Using the window.location for the origin is a flexible solution because the config (to whitelist the redirect uri) only needs to be added on the Okta side for each domain. It is common for people to take our sample and deploy it at a temporary URL. In these cases, they will need to update the config in Okta Admin UI, but (due to this change) they will not need to redeploy or edit the sample to include the temporary URL.

I think we would NOT want to encourage people to swap "localhost" with the name of their production domain in the source. They will need the app to run in localhost, production, and other hosting scenarios without code changes. I think hardcoding to "localhost" was showing a bad practice, and with this update we are demonstrating a better practice.